gatsby-config.js

```diff
@@ -139,6 +139,13 @@ const getPlugins = () => {
         useClassNames: true,
       },
     },
+    {
+      resolve: 'gatsby-plugin-google-tagmanager',
+      options: {
+        id: 'GTM-NXDBVV',
```

Comment:
@ozairashfaqueSSW 
Can you please make this a build pipeline parameter similar to `process.env.GOOGLE_ANALYTICS`
Maybe something like `GOOGLE_GTM_ID`

Reply:
Got it, i will do that now